src/main/java/reposense/git/GitCatFile.java

```diff
@@ -0,0 +1,57 @@
+package reposense.git;
+
+import static reposense.system.CommandRunner.runCommand;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+import reposense.git.exception.CommitNotFoundException;
+import reposense.system.LogsManager;
+
+/**
+ * Contains git cat file related functionalities.
+ * Git cat file provides content or type and size information for repository objects.
+ */
+public class GitCatFile {
+
+    private static final Logger logger = LogsManager.getLogger(GitCatFile.class);
+
+    /**
+     * Returns parent commits for the commit associated with the input commit hash.
+     */
+    public static List<String> getParentCommits(String root, String commitHash) throws CommitNotFoundException {
+        Path rootPath = Paths.get(root);
+        String catFileCommand = "git cat-file -p " + commitHash;
+        try {
+            String output = runCommand(rootPath, catFileCommand);
+            List<String> parents = new ArrayList();
```

Comment:
Can name `parentCommits` instead of `parents`?

Reply:
I have changed this.